apps/website/.vuepress/theme/styles/overrides.scss

```diff
@@ -252,3 +252,31 @@ h4 .header-anchor cds-icon {
     }
   }
 }
+
+/*
+ * This section is an override for something that doesn't exist in Core yet.
```

Comment:
We aren't adding tokens for this. Your implementation will remain the same in the HTML, however.

But there is no value in having these line heights be separate tokens when Existing tokens￼￼￼ can be updated in the CSS.￼

Reply:
That's not what I'm saying. I'm saying that we should be able to override our _existing_ CSS custom properties using CSS. What is the implementation difference between:

```
[cds-text*='expanded'][cds-text*='body'] {
  --cds-token-typography-body-line-height: 500rem;
}
```

...and...

```
:root {
  --cds-token-typography-body-line-height-expanded: 500rem;
}

[cds-text*='expanded'][cds-text*='body'] {
  line-height: var(--cds-token-typography-body-line-height-expanded);
}
```

In the former, we are overriding an existing CSS var. In the latter, we are creating an entirely new set of CSS vars that will add 54 new line-height tokens to our code.

The former, imo, is more maintainable. The difference in implementation details is trivial. But the effect of geometrically increasing our token CSS vars is not sustainable.

That is why I am advocating for the former.